### PR TITLE
add new features or changes

### PR DESCRIPTION
**Pull request by Love4sharma**

### CODE REVIEW

**Comment by Love4sharma** [comment]
added new features

**Review by Love4sharma — COMMENTED** [review]
merged successfully

File: index.html [hunk 1]
```diff
@@ -29,7 +29,7 @@
       </div>
 
       <nav>
-        <a href="#" class="btn btn-m">
```
Comment (Love4sharma) [inline index.html:32]: change a anchor link

File: index.html
Comment (Love4sharma) [inline index.html:1]: <a href="login.html" class="btn btn-updated">Go to Login Page</a>

File: index.html (hunk 1 above)
Comment (Love4sharma) [inline index.html:32]: Comment
Submit general feedback without explicit approval.

Approve
Pull request authors can’t approve their own pull request

Request changes
Pull request authors can’t request changes on their own pull request

**Review by Love4sharma — COMMENTED** [review]
all files arew viewd

**Comment by Love4sharma** [comment]
review of all files done

**Comment by Kushsharma2001** [comment]
merged done

**Pull request merged**